### ISSUE TITLE
Swipe left and right navigation doesn't work on the website

### ISSUE THREAD

**Issue by oanaOM** [exercism/exercism#6679]
Hi,

I noticed that on the main website it doesn't work to navigate between pages using left and right touchpad swipe. I've replicated this in the video below:

https://user-images.githubusercontent.com/15228388/227803360-d88b46d0-562c-4bcf-b606-6b8ef08b7f84.mp4

Is this a known behaviour? Looks like this is coming from this line of [code](https://github.com/exercism/website/blob/main/app/css/defaults.css#L11) and I would happily contribute to this if it's not ☺️

**Comment by SleeplessByte** [comment]
My guess is that this was added to make the background not scroll when a dialog opens, but it's not the "right way".

**Comment by iHiD** [comment]
@oanaOM Hey. Thanks for opening.
@SleeplessByte Thanks.
@dem4ron Could you take a look pls. What is the consequence of removing that line?

**Comment by oanaOM** [comment]
Cool, thanks for fixing this! I can confirm that it works fine for me 🚀 🙏
